testbed/tests/trace_test.go

```diff
@@ -271,10 +326,10 @@ func verifySingleSpan(
 			count++
 		}
 	}
-	assert.EqualValues(t, 1, count, "must receive one span")
+	assert.EqualValues(t, 1, count, "must receive one Jaeger span")
 }
 
-func TestTraceAttributesProcessor(t *testing.T) {
+func TestJaegerTraceAttributesProcessor(t *testing.T) {
```

Comment:
This change is against the intent of the test. The test is already intended to test different formats using subtests. Please rebase and you will see that now it has 2 subtests, each using different senders and receiver. Zipkin can be added to as another subtest.